package.json

```diff
@@ -7,8 +7,8 @@
     "preinstall": "[ \"$INIT_CWD\" != \"$PWD\" ] || npm_config_yes=true npx check-engine"
   },
   "engines": {
-    "node": "12.x",
-    "npm": "7.x || 8.x"
+    "node": ">=16",
+    "npm": ">=7"
   },
   "devDependencies": {
```

Comment:
Suggestion: since we're ultimately dependent on AWS infra (via Circle and Heroku) and since the latest version of Node AWS supports is 14 it's probably safer to err on the side of caution and specify the same version. 

I'd also recommend pinning both Node and npm with Volta